PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-lib/src/main/java/com/microsoft/azure/toolkit/intellij/common/component/Tree.java

```diff
@@ -142,6 +147,9 @@ private void refreshChildrenView() {
         @Override
         @AzureOperation(name = "user/common.load_children.node", params = "this.getLabel()")
         public synchronized void refreshChildren(boolean... incremental) {
+            if (getData() instanceof AbstractAzResource<?, ?, ?>) {
+                TreeUtils.expandResource(tree, (AbstractAzResource<?, ?, ?>) getData());
```

Comment:
I guess this method should be renamed, it's strange that we need to expand the resource when refreshing its children.